### ISSUE TITLE
Partial import from namespace not detected

### ISSUE THREAD

**Issue by hvelarde** [reinout/z3c.dependencychecker#16]
In one of our packages I have plone.api listed as a dependency in the setup.py file and I use it like this in the code:

``` python
from plone import api
```

When I run dependencychecker, I get:

```
...
Unneeded requirements
=====================
     plone.api
...
```

**Comment by reinout** [comment]
I think this is caused by the issue identified here: https://github.com/reinout/z3c.dependencychecker/issues/23#issuecomment-135778075

This is fixed in release 1.13. If the issue is still relevant for you, could you check if it is fixed?

**Comment by gforcada** [comment]
Closing, reopen if it is still an issue, but the upcoming rewrite does not suffer from this problem either.
